components/board/numato/board.go

```diff
@@ -5,7 +5,6 @@ import (
 	"bufio"
 	"context"
 	"encoding/hex"
-	"errors"
```

Comment:
this package dependency was changed when you imported the new errors package. The APIs from the "https://github.com/pkg/errors" library may behave differently, let's use a "fmt" error in the one line that needs it in this file and revert to using the "errors" package.